testing/iprutils/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=iprutils
+pkgver=2.4.14
+pkgrel=0
+pkgdesc="user space tools required by the ipr device driver."
+url="https://github.com/bjking1/iprutils"
+arch="all"
+license="CPL"
+depends="automake libtool m4 autoconf ncurses-dev zlib-dev"
+makedepends=""
+install=
+subpackages="$pkgname-doc"
+_pkgver=$(echo $pkgver | sed 's/\./-/g')
+source="$pkgname-$pkgver.tar.gz::https://github.com/bjking1/$pkgname/archive/rel-$_pkgver.tar.gz"
+
+_builddir="$srcdir"/iprutils-rel-$_pkgver
+
+build() {
+	cd "$_builddir"
+	./bootstrap.sh
+
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--localstatedir=/var \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$_builddir"
+	make DESTDIR="$pkgdir" install
+	rm -rf "$pkgdir"/usr/lib
+	return 0
```

Comment:
`return 0` is not really needed, by default the status of last command is returned, which should be zero when successful.